release/1-19/release-announcement.txt

```diff
@@ -0,0 +1 @@
+Amazon EKS Distro now supports Kubernetes version 1.19. Highlights of the Kubernetes 1.19 release include Ingress API and Pod Topology Spread reaching stable status, EndpointSlices being enabled by default, and immutable Secrets and ConfigMaps. Amazon EKS Distro builds of Kubernetes 1.19 are available through ECR Public Gallery and GitHub.
```

Comment:
If you put this announcement in another PR, that will simplify things. Also we should have a url in the announcement probably so people have something to give them more context like a link to https://distro.eks.amazonaws.com/releases/1-19/1/ for this one